.travis.yml

```diff
@@ -99,7 +99,15 @@ script:
       echo "Skipping script build step in Coverity branch/job";
     else
       travis_retry travis_wait 50 \
-          mvn clean verify --projects janusgraph-${MODULE} ${ARGS};
+          mvn clean verify --projects janusgraph-${MODULE} -Pcoverage ${ARGS};
+    fi
+
+after_success:
```

Comment:
@mbrukman The code coverage analysis happens during testing (through JaCoCo instrumentation) and all that's uploaded is the final report, which will just be for the given module in the Travis job. Codecov handles the aggregation of the reports.

https://github.com/codecov/support/wiki/Merging-Reports